File: VagasWEB/test/testVagas.java
import static org.junit.Assert.assertEquals;

import org.junit.Test;

import fspoa.vagas.Curso;

public class testVagas {

	/**
	 * 
	 * Teste que verifica se o ID do curso está de acordo com o pesquisado.
	 * 
	 */

	@Test
	public void testIdCurso() {

		Curso curso = new Curso();

		curso.setId(34);

		assertEquals(34, curso.getId());

		curso.setId(1);

		assertEquals(1, curso.getId());

	}

	/**
	 * Teste que verifica se o ID do curso e ID da cidade está de acordo com o
	 * pesquisado.
	 * 
	 */

	@Test
	public void testCursoIdCidadeId() {

		Curso curso = new Curso();

		Cidade cidade = new Cidade();

		curso.setId(1);

		cidade.setId(35);
		
		assertEquals(35, cidade.getId());

		assertEquals(1, curso.getId());

	}
	
	@Test
	public void testCursoNegativo(){
		Curso curso = new Curso();
		curso.setId(-15);
		
		assertEquals(0, curso.getId());
	}

}

File: VagasWEB/test/testVagas.php
<?php
	

	/*

	1.1. entrada: (cursoId) [34] OK
	1.2. entrada: (cursoId) [1] OK
	1.3. entrada: (cursoId, cidadeId) [1, 35]

	Operação inválidas

	1.1. entrada: (curso) [-15] 
	1.2. entrada: (curso) [Recurso Humano] 
	1.3. entrada: (cursoId, cidadeId) [1, 0]
	*/

	class testVagas extends PHPUnit_Framework_TestCase{

	

	/*
	* Teste verifica se o ID do curso está de acordo com o pesquisado
	*/

		public function testIdCurso(){
			
			include '../curso.php';

			$curso = new Curso();

			$curso.setId(34);

			$this -> assertEquals(34, $curso->getId());

		}	

	}


?>